x-pack/plugins/cases/server/client/metrics/all_cases/aggregations/status_counts.test.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CasePersistedStatus } from '../../../../common/types/case';
+import { StatusCounts } from './status_counts';
+
+describe('StatusCounts', () => {
+  it('returns the correct aggregation', async () => {
```

Comment:
you can remove all async usages in this file

Reply:
Done in [`69447f9` (#201098)](https://github.com/elastic/kibana/pull/201098/commits/69447f9065a127cdb5e40ea970a5ece4a3315ec4).